docs/overview/changelog.md

```diff
@@ -7,6 +7,34 @@ up-to-date list of all changes.
 
 ## Zulip Server 9.x series
 
+### Zulip Server 9.2
+
+_Released 2024-09-12_
+
+- Fixed size of EDITED/MOVED indicators when not using compact mode.
+- Fixed Firefox being installed via an indirect `apt` recommendation.
```

Comment:
I don't _think_ it's worth clarifying that this does not remove the package if it has been installed due ti installing 9.0 or 9.1 -- because it reads a bit like we do remove it.  Which would be nice, but isn't what we did.